src/opm/parser/eclipse/EclipseState/Tables/TableManager.cpp

```diff
@@ -442,6 +442,7 @@ DensityTable make_density_table(const GravityTable& gravity) {
         addTables( "PDVD", m_eqldims.getNumEquilRegions());
         addTables( "SALTVD", m_eqldims.getNumEquilRegions());
         addTables( "SALTPVD", m_eqldims.getNumEquilRegions());
+        addTables( "SALTSOL", m_eqldims.getNumEquilRegions());
```

Comment:
The JSON schema suggests that the number of tables/records is the declared number of PVT tables ("NTPVT").  Did you intend to say
```C++
addTables("SALTSOL", m_tabdims.getNumPVTTables())
```
here instead?